### PR TITLE
Kubeflow autodeploy jobs need to use kubeflow/kfctl

### PR DESCRIPTION
**Pull request by jlewi**
* Related to kubeflow/kfctl#7

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/522)

### CODE REVIEW

File: test-infra/auto-deploy/deploy-master.yaml [hunk 1]
```diff
@@ -20,8 +20,8 @@ spec:
       initContainers:
       - command:        
         - /usr/local/bin/checkout_repos.sh 
-        # Stop using PR #495 once its subbmitted
-        - --repos=kubeflow/kubeflow@HEAD,kubeflow/testing@HEAD:506
+        # Stop using PR #495 once its subbmitted
```
Comment (zhenghuiwang) [inline test-infra/auto-deploy/deploy-master.yaml]: Should we remove this comment and not pin to `testing@HEAD:506` in this PR?

Comment (jlewi) [reply test-infra/auto-deploy/deploy-master.yaml]: Fixed

**Pull request merged**